obspy/io/nordic/core.py

```diff
@@ -1280,7 +1282,30 @@ def _write_hyp_error_line(origin):
         errors = origin.origin_uncertainty[
             'confidence_ellipsoid'].confidence_ellipsoid_to_xyz()
     except AttributeError:
-        return ''.join(error_line)
+        try:
+            cov = Ellipse.from_origin_uncertainty(origin.origin_uncertainty).\
```

Comment:
To avoid all this nesting within try excepts, could you just work out whether the attribute is there or not and select the correct method? Otherwise this seems like quite a bit of duplication.

Reply:
OK, I changed this, which actually also laid bare that the first case  uses an inexistent function (confidence_ellipsoid_to_xyz())!  This first part, which depends on having an Ellipsoid class, is now commented out.  I've actually written the Ellipsoid class, but embedding it is a lot more work and changes than this bugfix deserves.